Kwenta/kwenta
# Handle network changes

These changes handle network changes between Optimism mainnet and Optimism Kovan testnet more gracefully. 

## Description
* Add query keys where necessary to refresh on network changes
* Update queries to import network state
* Update hooks to choose the network based on `network.id`, defaulting to Optimism Mainnet

## Related issue
Closes https://github.com/Kwenta/kwenta-subgraph/issues/4

## Motivation and Context
Currently data is disconnected between network selected in metamask, and often requires refreshing to get data from the expected network.

## How Has This Been Tested?
* Connect wallet
* View dashboard or markets page
* Switch wallet between mainnet and testnet
* Verify that all tables refresh
